src/workflows-app/SubmissionConfig.test.js

```diff
@@ -419,30 +420,1486 @@ describe('BaseSubmissionConfig with workflowsAppStore', () => {
     });
 
     // ** ACT **
-    render(
-      h(BaseSubmissionConfig, {
-        methodId: '123',
-        name: 'test-azure-ws-name',
-        namespace: 'test-azure-ws-namespace',
-        workspace: mockAzureWorkspace,
-      })
+    await act(async () =>
+      render(
+        h(BaseSubmissionConfig, {
+          methodId: '123',
+          name: 'test-azure-ws-name',
+          namespace: 'test-azure-ws-namespace',
+          workspace: mockAzureWorkspace,
+        })
+      )
     );
 
     // ** ASSERT **
-    await waitFor(() => {
-      expect(mockRunSetResponse).toHaveBeenCalledTimes(1);
-      expect(mockTypesResponse).toHaveBeenCalledTimes(1);
-      expect(mockMethodsResponse).toHaveBeenCalledTimes(1);
-      expect(mockSearchResponse).toHaveBeenCalledTimes(1);
-      expect(mockWdlResponse).toHaveBeenCalledTimes(1);
-      // Leo is called since proxy urls for WDS is not available yet
-      expect(mockLeoResponse).toHaveBeenCalledTimes(1);
+
+    expect(mockRunSetResponse).toHaveBeenCalledTimes(1);
+    expect(mockTypesResponse).toHaveBeenCalledTimes(1);
+    expect(mockMethodsResponse).toHaveBeenCalledTimes(1);
+    expect(mockSearchResponse).toHaveBeenCalledTimes(1);
+    expect(mockWdlResponse).toHaveBeenCalledTimes(1);
+    // Leo is called since proxy urls for WDS is not available yet
+    expect(mockLeoResponse).toHaveBeenCalledTimes(1);
+
+    // verify that 'wdsProxyUrlState' in store was updated
+    expect(workflowsAppStore.get().wdsProxyUrlState).toStrictEqual({
+      status: AppProxyUrlStatus.Ready,
+      state: 'https://lz-abc/wds-abc-c07807929cd1/',
+    });
+  });
+
+  it('should call Leo to get proxy url for WDS if its not ready', async () => {
```

Comment:
Is this a duplicated test or is Github playing tricks with the diff? 🤔

Reply:
Oops, I think I goofed up the merge here.